common/djangoapps/student/helpers.py

```diff
@@ -270,6 +262,31 @@ def get_next_url_for_login_page(request):
     return redirect_to
 
 
+def get_redirect_to(request):
+    """
+    Determine the redirect url and return if safe
+    :argument
+        request: request object
+
+    :returns: redirect url if safe else None
+    """
+    static_file_format = '\w+\.(jpeg|jpg|gif|png|ico)'
```

Comment:
Is that necessarily true for all assets? Are you confident that we have no extension-less assets in the plaftorm and that course teams can't upload assets without extensions?